setup_rucio.py

```diff
@@ -82,13 +82,13 @@ def run_git_command(cmd):
 
 if using_pip and using_git:
     git_version_cmd = '''git describe --dirty=-dev`date +%s`'''
-    pkg_version = run_git_command(git_version_cmd).decode()
+    pkg_version = run_git_command(git_version_cmd)
```

Comment:
It should be better to add the `decode()` call directly in the `run_git_command` function.

Reply:
I don't think it is needed at all. It was removed from the `tools/generate_version.py` and it runs fine. In principle I think this if branch can be removed completely, since we always create the `vcsversion.py` file by hand anyway.

Reply:
right, I overlooked the change and think it was the other way around.. brrrrrr